### DISCUSSION TITLE
[Config Support]: Reolink WiFi Doorbell 2 Way audio

### DISCUSSION THREAD

**Discussion by Nooton92** [blakeblackshear/frigate#14737] [Config Support]
### Describe the problem you are having

I am having trouble getting my two way audio working both in frigate and in HA. I have audio coming from the doorbell to my phone just fine and even have the enable mic option. If I press the enable mic option the camera feed reloads and I lose all audio from the doorbell to my phone and I get no audio from my phone to the doorbell either.

My camera is up to date on firmware (Oct 23rd I believe was the newest one). Running Frigate as a docker container in UnRaid(also up to date). Using my CPU for now....(coral TPU in on the way). Reverse proxy with Nginx.

Trying to get the audio working in the Frigate app first. Then I will worry about HA.

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
ffmpeg:
  input_args: preset-rtsp-generic
  output_args:
    record: preset-record-generic-audio-copy
go2rtc:
  streams:
    DoorCam:
      - ffmpeg:http://<IP>/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=<USER>&password=<PASS>#audio=volume#audio=opus#audio=copy
      - rtsp://<USER>:<PASS>@<IP>:554/Preview_01_sub
    DoorCam_sub:
      - ffmpeg:http://<IP>/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=<USER>&password=<PASS>#audio=volume#audio=opus#audio=copy
      - rtsp://<USER>:<PASS@<IP>:554/h264Preview_01_sub
    DrivewayCam:
      - ffmpeg:rtsp://<USER>:<PASS@<IP>Preview_01_main
    DrivewayCam_sub:
      - ffmpeg:rtsp://<USER>:<PASS@<IP>@<IP>/Preview_01_sub
  ffmpeg:
    bin: ffmpeg
    volume: -af "volume=30dB"
  webrtc:
    candidates:
      - <Server IP>:8555
      - stun:8555
```

### Relevant Frigate log output

```shell
2024-11-02 12:09:36.821009195  [INFO] Preparing Frigate...
2024-11-02 12:09:36.866741293  [INFO] Starting Frigate...
2024-11-02 12:09:41.344467741  [2024-11-02 12:09:41] frigate.app                    INFO    : Starting Frigate (0.14.1-f4f3cfa)
2024-11-02 12:09:41.344819363  [2024-11-02 12:09:41] frigate.util.config            INFO    : Checking if frigate config needs migration...
2024-11-02 12:09:41.404089744  [2024-11-02 12:09:41] frigate.util.config            INFO    : frigate config does not need migration...
2024-11-02 12:09:42.931898656  [2024-11-02 12:09:42] frigate.util.services          INFO    : Automatically detected nvidia hwaccel for video decoding
2024-11-02 12:09:43.016328908  [2024-11-02 12:09:43] peewee_migrate.logs            INFO    : Starting migrations
2024-11-02 12:09:43.017634847  [2024-11-02 12:09:43] peewee_migrate.logs            INFO    : There is nothing to migrate
2024-11-02 12:09:43.050222646  [2024-11-02 12:09:43] frigate.app                    INFO    : Recording process started: 417
2024-11-02 12:09:43.065478271  [2024-11-02 12:09:43] frigate.app                    INFO    : Recording process started: 418
2024-11-02 12:09:43.074728241  [2024-11-02 12:09:43] frigate.app                    INFO    : go2rtc process pid: 117
2024-11-02 12:09:43.128126676  [2024-11-02 12:09:43] detector.cpu                   INFO    : Starting detection process: 446
2024-11-02 12:09:43.133211748  [2024-11-02 12:09:43] frigate.detectors              WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
2024-11-02 12:09:43.135051077  [2024-11-02 12:09:43] frigate.app                    INFO    : Output process started: 448
2024-11-02 12:09:43.180279593  [2024-11-02 12:09:43] frigate.app                    INFO    : Camera processor started for DoorCam: 462
2024-11-02 12:09:43.188715509  [2024-11-02 12:09:43] frigate.app                    INFO    : Camera processor started for DrivewayCam: 463
2024-11-02 12:09:43.214379282  [2024-11-02 12:09:43] frigate.app                    INFO    : Capture process started for DoorCam: 464
2024-11-02 12:09:43.231301351  [2024-11-02 12:09:43] frigate.app                    INFO    : Capture process started for DrivewayCam: 468
```

### Relevant go2rtc log output

```shell
2024-11-02 12:09:36.823165985  [INFO] Preparing new go2rtc config...
2024-11-02 12:09:37.825201974  [INFO] Not injecting WebRTC candidates into go2rtc config as it has been set manually
2024-11-02 12:09:37.898452133  [WARN] Using go2rtc binary from '/config/go2rtc' instead of the embedded one
2024-11-02 12:09:37.898474484  [INFO] Starting go2rtc...
2024-11-02 12:09:38.004204602  12:09:38.004 INF go2rtc version 1.8.5 linux/amd64
2024-11-02 12:09:38.004952020  12:09:38.004 INF [api] listen addr=:1984
2024-11-02 12:09:38.005462050  12:09:38.005 INF [rtsp] listen addr=:8554
2024-11-02 12:09:38.006175167  12:09:38.006 INF [webrtc] listen addr=:8555
2024-11-02 12:09:46.822092642  [INFO] Starting go2rtc healthcheck service...
```

### Frigate stats

```json
{"cameras":{"DoorCam":{"audio_dBFS":0.0,"audio_rms":0.0,"camera_fps":5.1,"capture_pid":464,"detection_enabled":true,"detection_fps":0.0,"ffmpeg_pid":470,"pid":462,"process_fps":5.1,"skipped_fps":0.0},"DrivewayCam":{"audio_dBFS":0.0,"audio_rms":0.0,"camera_fps":5.1,"capture_pid":468,"detection_enabled":true,"detection_fps":1.0,"ffmpeg_pid":494,"pid":463,"process_fps":5.1,"skipped_fps":0.0}},"cpu_usages":{"1":{"cmdline":"/package/admin/s6/command/s6-svscan -d4 -- /run/service","cpu":"0.0","cpu_average":"0","mem":"0.0"},"106":{"cmdline":"s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/nginx","cpu":"0.0","cpu_average":"0","mem":"0.0"},"107":{"cmdline":"s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/frigate","cpu":"0.0","cpu_average":"0","mem":"0.0"},"108":{"cmdline":"s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/go2rtc","cpu":"0.0","cpu_average":"0","mem":"0.0"},"109":{"cmdline":"s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/certsync","cpu":"0.0","cpu_average":"0","mem":"0.0"},"117":{"cmdline":"/config/go2rtc -config=/dev/shm/go2rtc.yaml","cpu":"2.8","cpu_average":"2","mem":"0.1"},"132":{"cmdline":"bash ./run.user go2rtc-healthcheck","cpu":"0.0","cpu_average":"0","mem":"0.0"},"134":{"cmdline":"python3 -u -m frigate","cpu":"1.7","cpu_average":"2","mem":"2.7"},"151":{"cmdline":"nginx: master process nginx","cpu":"0.0","cpu_average":"0","mem":"0.1"},"1627":{"cmdline":"sleep 60","cpu":"0.0","cpu_average":"0","mem":"0.0"},"1750":{"cmdline":"sleep 30s","cpu":"0.0","cpu_average":"0","mem":"0.0"},"178":{"cmdline":"nginx: worker process","cpu":"0.0","cpu_average":"0","mem":"0.1"},"179":{"cmdline":"nginx: worker process","cpu":"0.0","cpu_average":"0","mem":"0.0"},"180":{"cmdline":"nginx: worker process","cpu":"0.0","cpu_average":"0","mem":"0.1"},"183":{"cmdline":"nginx: worker process","cpu":"0.1","cpu_average":"0","mem":"0.0"},"194":{"cmdline":"nginx: cache manager process","cpu":"0.0","cpu_average":"0","mem":"0.0"},"27":{"cmdline":"s6-supervise s6-linux-init-shutdownd","cpu":"0.0","cpu_average":"0","mem":"0.0"},"28":{"cmdline":"/package/admin/s6-linux-init/command/s6-linux-init-shutdownd -c /run/s6/basedir -g 3000 -C -B","cpu":"0.0","cpu_average":"0","mem":"0.0"},"315":{"cmdline":"bash ./run.user certsync","cpu":"0.0","cpu_average":"0","mem":"0.0"},"36":{"cmdline":"s6-supervise s6rc-oneshot-runner","cpu":"0.0","cpu_average":"0","mem":"0.0"},"369":{"cmdline":"frigate.logger       ","cpu":"0.0","cpu_average":"0","mem":"1.2"},"37":{"cmdline":"s6-supervise s6rc-fdholder","cpu":"0.0","cpu_average":"0","mem":"0.0"},"38":{"cmdline":"s6-supervise certsync","cpu":"0.0","cpu_average":"0","mem":"0.0"},"39":{"cmdline":"s6-supervise certsync-log","cpu":"0.0","cpu_average":"0","mem":"0.0"},"40":{"cmdline":"s6-supervise frigate","cpu":"0.0","cpu_average":"0","mem":"0.0"},"41":{"cmdline":"s6-supervise frigate-log","cpu":"0.0","cpu_average":"0","mem":"0.0"},"417":{"cmdline":"frigate.recording_manager","cpu":"0.5","cpu_average":"0","mem":"1.3"},"418":{"cmdline":"frigate.review_segment_manager","cpu":"0.3","cpu_average":"0","mem":"1.2"},"42":{"cmdline":"s6-supervise go2rtc","cpu":"0.0","cpu_average":"0","mem":"0.0"},"43":{"cmdline":"s6-supervise go2rtc-healthcheck","cpu":"0.0","cpu_average":"0","mem":"0.0"},"44":{"cmdline":"s6-supervise go2rtc-log","cpu":"0.0","cpu_average":"0","mem":"0.0"},"445":{"cmdline":"/usr/bin/python3 -c from multiprocessing.resource_tracker import main;main(60)","cpu":"0.2","cpu_average":"0","mem":"0.1"},"446":{"cmdline":"frigate.detector.cpu ","cpu":"50.1","cpu_average":"37","mem":"1.3"},"448":{"cmdline":"frigate.output       ","cpu":"0.7","cpu_average":"0","mem":"1.3"},"45":{"cmdline":"s6-supervise nginx","cpu":"0.0","cpu_average":"0","mem":"0.0"},"46":{"cmdline":"s6-supervise nginx-log","cpu":"0.0","cpu_average":"0","mem":"0.0"},"462":{"cmdline":"frigate.process:DoorCam","cpu":"1.5","cpu_average":"1","mem":"1.3"},"463":{"cmdline":"frigate.process:DrivewayCam","cpu":"5.5","cpu_average":"4","mem":"1.3"},"464":{"cmdline":"frigate.capture:DoorCam","cpu":"1.8","cpu_average":"1","mem":"1.2"},"467":{"cmdline":"ffmpeg -threads 1 -f rawvideo -pix_fmt yuv420p -video_size 1280x720 -i pipe: -threads 1 -f mpegts -s 1280x720 -codec:v mpeg1video -q 8 -bf 0 pipe:","cpu":"0.0","cpu_average":"0","mem":"0.1"},"468":{"cmdline":"frigate.capture:DrivewayCam","cpu":"0.9","cpu_average":"0","mem":"1.2"},"470":{"cmdline":"ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel cuda -hwaccel_output_format cuda -user_agent FFmpeg Frigate/0.14.1-f4f3cfa -rtsp_transport tcp -timeout 5000000 -i rtsp://127.0.0.1:8554/DoorCam_sub?video=copy -r 5 -vf fps=5,scale_cuda=w=1280:h=720:format=nv12,hwdownload,format=nv12,format=yuv420p -threads 2 -f rawvideo -pix_fmt yuv420p pipe:","cpu":"2.0","cpu_average":"1","mem":"0.7"},"494":{"cmdline":"ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel cuda -hwaccel_output_format cuda -user_agent FFmpeg Frigate/0.14.1-f4f3cfa -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -timeout 5000000 -use_wallclock_as_timestamps 1 -i rtsp://127.0.0.1:8554/DrivewayCam_sub -r 5 -vf fps=5,scale_cuda=w=640:h=360:format=nv12,hwdownload,format=nv12,format=yuv420p -threads 2 -f rawvideo -pix_fmt yuv420p pipe:","cpu":"1.1","cpu_average":"0","mem":"0.6"},"499":{"cmdline":"ffmpeg -threads 1 -f rawvideo -pix_fmt yuv420p -video_size 640x360 -i pipe: -threads 1 -f mpegts -s 1280x720 -codec:v mpeg1video -q 8 -bf 0 pipe:","cpu":"0.0","cpu_average":"0","mem":"0.1"},"503":{"cmdline":"ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel cuda -hwaccel_output_format cuda -user_agent FFmpeg Frigate/0.14.1-f4f3cfa -rtsp_transport tcp -timeout 5000000 -i rtsp://127.0.0.1:8554/DoorCam?video=copy&audio=aac -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy /tmp/cache/DoorCam@%Y%m%d%H%M%S%z.mp4","cpu":"0.3","cpu_average":"0","mem":"0.1"},"504":{"cmdline":"ffmpeg -hide_banner -v error -allowed_media_types video+audio -fflags nobuffer -flags low_delay -timeout 5000000 -user_agent go2rtc/ffmpeg -rtsp_flags prefer_tcp -i rtsp://*:*@192.168.2.100/Preview_01_sub -c copy -user_agent ffmpeg/go2rtc -rtsp_transport tcp -f rtsp rtsp://127.0.0.1:8554/2afbd31fb54121e5a52a0a28c724ffd3","cpu":"0.6","cpu_average":"0","mem":"0.1"},"505":{"cmdline":"ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel cuda -hwaccel_output_format cuda -user_agent FFmpeg Frigate/0.14.1-f4f3cfa -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -timeout 5000000 -use_wallclock_as_timestamps 1 -i rtsp://127.0.0.1:8554/DrivewayCam -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy /tmp/cache/DrivewayCam@%Y%m%d%H%M%S%z.mp4","cpu":"1.3","cpu_average":"0","mem":"0.1"},"512":{"cmdline":"ffmpeg -hide_banner -v error -allowed_media_types video+audio -fflags nobuffer -flags low_delay -timeout 5000000 -user_agent go2rtc/ffmpeg -rtsp_flags prefer_tcp -i rtsp://*:*@<Camera IP>/Preview_01_main -c copy -user_agent ffmpeg/go2rtc -rtsp_transport tcp -f rtsp rtsp://127.0.0.1:8554/59fd5c516fb87b1a19084d9457f8d5bb","cpu":"3.7","cpu_average":"3","mem":"0.1"},"513":{"cmdline":"ffmpeg -threads 1 -f rawvideo -pix_fmt yuv420p -video_size 1280x720 -i pipe: -threads 1 -f mpegts -s 1280x720 -codec:v mpeg1video -q 8 -bf 0 pipe:","cpu":"0.0","cpu_average":"0","mem":"0.1"},"57":{"cmdline":"/package/admin/s6/command/s6-ipcserverd -1 -- /package/admin/s6/command/s6-ipcserver-access -v0 -E -l0 -i data/rules -- /package/admin/s6/command/s6-sudod -t 30000 -- /package/admin/s6-rc/command/s6-rc-oneshot-run -l ../.. --","cpu":"0.0","cpu_average":"0","mem":"0.0"},"58":{"cmdline":"/package/admin/s6-2.11.3.2/command/s6-fdholderd -1 -i data/rules","cpu":"0.0","cpu_average":"0","mem":"0.0"},"frigate.full_system":{"cpu":"14.5","mem":"44.0"}},"detection_fps":1.0,"detectors":{"cpu":{"detection_start":0.0,"inference_speed":65.92,"pid":446}},"gpu_usages":{"Quadro K620":{"dec":"1.0%","enc":"0.0%","gpu":"0.0%","mem":"8.57%"}},"processes":{"go2rtc":{"pid":117},"logger":{"pid":369},"recording":{"pid":417},"review_segment":{"pid":418}},"service":{"last_updated":1730571324,"latest_version":"0.14.1","storage":{"/dev/shm":{"free":253.0,"mount_type":"tmpfs","total":256.0,"used":3.0},"/media/frigate/clips":{"free":16861013.6,"mount_type":"fuse.shfs","total":38144067.2,"used":21283053.6},"/media/frigate/recordings":{"free":16861013.6,"mount_type":"fuse.shfs","total":38144067.2,"used":21283053.6},"/tmp/cache":{"free":946.9,"mount_type":"tmpfs","total":953.7,"used":6.8}},"temperatures":{},"uptime":341,"version":"0.14.1-f4f3cfa"}}
```

### Operating system

HassOS

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
Sorry...not sure how to get this in UnRaid. If required please provide guidance.
```

### Object Detector

CPU (no coral)

### Screenshots of the Frigate UI's System metrics pages

![Camera System Metrics](https://github.com/user-attachments/assets/43964c33-dcf3-40e8-81de-e1577fe4075e)
![General System Metrics](https://github.com/user-attachments/assets/d562db50-b15c-412e-a48e-ad6be885e979)

**Comment by NickM-27** [comment]
it won't work in frigate because you are using an old version of go2rtc, you should delete the manual version and let frigate use the default

**Reply by Nooton92** [reply]
Ok I did as you said. Still same issue. Audio to my phone works until I enable the mic then no audio either direction.

2024-11-02 12:57:11.784900578  [INFO] Preparing new go2rtc config...
2024-11-02 12:57:12.707126537  [INFO] Not injecting WebRTC candidates into go2rtc config as it has been set manually
2024-11-02 12:57:12.791100737  [INFO] Starting go2rtc...
2024-11-02 12:57:12.996043449  12:57:12.995 INF go2rtc platform=linux/amd64 revision=b2399f3 version=1.9.2
2024-11-02 12:57:12.997594652  12:57:12.996 INF config path=/dev/shm/go2rtc.yaml
2024-11-02 12:57:12.997598201  12:57:12.996 INF [rtsp] listen addr=:8554
2024-11-02 12:57:12.997604562  12:57:12.997 INF [webrtc] listen addr=:8555
2024-11-02 12:57:12.998591526  12:57:12.998 INF [api] listen addr=:1984
2024-11-02 12:57:21.782723710  [INFO] Starting go2rtc healthcheck service...

**Reply by NickM-27** [reply]
we will need to see the contents of the probe when opening the camera live view in frigate, using the network browser tool
<img width="730" alt="Screenshot 2024-11-02 at 1 17 53 PM" src="https://github.com/user-attachments/assets/7384e923-f624-45b3-b169-8b7907bc5621">

**Reply by Nooton92** [reply]
Here is with just audio(mic not activated yet):
{
    "consumers": [
        {
            "medias": [
                "audio, sendonly, MPEG4-GENERIC",
                "video, sendonly, ANY"
            ],
            "remote_addr": "127.0.0.1:52766",
            "sdp": "v=0\r\no=- 1 1 IN IP4 0.0.0.0\r\ns=go2rtc/1.9.2\r\nc=IN IP4 0.0.0.0\r\nt=0 0\r\nm=audio 0 RTP/AVP 96\r\na=rtpmap:96 MPEG4-GENERIC/16000\r\na=fmtp:96 streamtype=5;profile-level-id=1;mode=AAC-hbr;sizelength=13;indexlength=3;indexdeltalength=3;config=1408\r\na=control:trackID=0\r\nm=video 0 RTP/AVP 97\r\na=rtpmap:97 H264/90000\r\na=fmtp:97 packetization-mode=1;profile-level-id=640033;sprop-parameter-sets=Z2QAM6wVFKCgPZA=,aO48sA==\r\na=control:trackID=1\r\n",
            "send": 361273972,
            "senders": [
                "96 MPEG4-GENERIC/16000, bytes=38046072, receivers=1",
                "97 H264, bytes=315295676, receivers=1"
            ],
            "type": "RTSP passive consumer",
            "url": "rtsp://127.0.0.1:8554/DoorCam?video=copy&audio=aac",
            "user_agent": "FFmpeg Frigate/0.14.1-f4f3cfa"
        },
        {
            "medias": [
                "video, sendonly, ALL",
                "audio, sendonly, ALL",
                "audio, recvonly, ANY"
            ],
            "receivers": [
                "0 ANY, bytes=0, senders=1"
            ],
            "remote_addr": "127.0.0.1:45178",
            "senders": [
                "96 H264, bytes=0, receivers=1",
                "97 MPEG4-GENERIC/16000, bytes=0, receivers=1"
            ],
            "type": "probe",
            "user_agent": "python-requests/2.32.3"
        }
    ],
    "producers": [
        {
            "url": "ffmpeg:http://192.168.2.101/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=*&password=*"
        },
        {
            "medias": [
                "video, recvonly, H.264 High 5.1",
                "audio, recvonly, MPEG4-GENERIC/16000",
                "audio, sendonly, PCMU/8000"
            ],
            "receivers": [
                "97 MPEG4-GENERIC/16000, bytes=38046072, senders=2",
                "96 H264, bytes=315295676, senders=2"
            ],
            "recv": 359475340,
            "remote_addr": "192.168.2.101:554",
            "sdp": "v=0\r\no=- 1730624460385733 1 IN IP4 192.168.2.101\r\ns=Session streamed by \"preview\"\r\nt=0 0\r\na=tool:BC Streaming Media v202210012022.10.01\r\na=type:broadcast\r\na=control:*\r\na=range:npt=now-\r\na=x-qt-text-nam:Session streamed by \"preview\"\r\nm=video 0 RTP/AVP 96\r\nc=IN IP4 0.0.0.0\r\nb=AS:8192\r\na=rtpmap:96 H264/90000\r\na=fmtp:96 packetization-mode=1;profile-level-id=640033;sprop-parameter-sets=Z2QAM6wVFKCgPZA=,aO48sA==\r\na=recvonly\r\na=control:track1\r\nm=audio 0 RTP/AVP 97\r\nc=IN IP4 0.0.0.0\r\nb=AS:8192\r\na=rtpmap:97 MPEG4-GENERIC/16000\r\na=fmtp:97 streamtype=5;profile-level-id=1;mode=AAC-hbr;sizelength=13;indexlength=3;indexdeltalength=3;config=1408\r\na=recvonly\r\na=control:track2\r\nm=audio 0 RTP/AVP 0\r\na=control:track3\r\na=rtpmap:0 PCMU/8000\r\na=sendonly",
            "senders": [
                "0 PCMU/8000, bytes=0, receivers=1"
            ],
            "type": "RTSP active producer",
            "url": "rtsp://192.168.2.101/Preview_01_sub/",
            "user_agent": "go2rtc/1.9.2"
        }
    ]
}

Here is with the mic activated:

{
    "consumers": [
        {
            "medias": [
                "audio, sendonly, MPEG4-GENERIC",
                "video, sendonly, ANY"
            ],
            "remote_addr": "127.0.0.1:52766",
            "sdp": "v=0\r\no=- 1 1 IN IP4 0.0.0.0\r\ns=go2rtc/1.9.2\r\nc=IN IP4 0.0.0.0\r\nt=0 0\r\nm=audio 0 RTP/AVP 96\r\na=rtpmap:96 MPEG4-GENERIC/16000\r\na=fmtp:96 streamtype=5;profile-level-id=1;mode=AAC-hbr;sizelength=13;indexlength=3;indexdeltalength=3;config=1408\r\na=control:trackID=0\r\nm=video 0 RTP/AVP 97\r\na=rtpmap:97 H264/90000\r\na=fmtp:97 packetization-mode=1;profile-level-id=640033;sprop-parameter-sets=Z2QAM6wVFKCgPZA=,aO48sA==\r\na=control:trackID=1\r\n",
            "send": 369038147,
            "senders": [
                "96 MPEG4-GENERIC/16000, bytes=38864125, receivers=1",
                "97 H264, bytes=322074166, receivers=1"
            ],
            "type": "RTSP passive consumer",
            "url": "rtsp://127.0.0.1:8554/DoorCam?video=copy&audio=aac",
            "user_agent": "FFmpeg Frigate/0.14.1-f4f3cfa"
        },
        {
            "medias": [
                "video, sendonly, ALL",
                "audio, sendonly, ALL",
                "audio, recvonly, ANY"
            ],
            "receivers": [
                "0 ANY, bytes=0, senders=1"
            ],
            "remote_addr": "127.0.0.1:56580",
            "senders": [
                "96 H264, bytes=0, receivers=1",
                "97 MPEG4-GENERIC/16000, bytes=0, receivers=1"
            ],
            "type": "probe",
            "user_agent": "python-requests/2.32.3"
        }
    ],
    "producers": [
        {
            "url": "ffmpeg:http://192.168.2.101/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=*&password=*"
        },
        {
            "medias": [
                "video, recvonly, H.264 High 5.1",
                "audio, recvonly, MPEG4-GENERIC/16000",
                "audio, sendonly, PCMU/8000"
            ],
            "receivers": [
                "97 MPEG4-GENERIC/16000, bytes=38864125, senders=2",
                "96 H264, bytes=322074166, senders=2"
            ],
            "recv": 367202795,
            "remote_addr": "192.168.2.101:554",
            "sdp": "v=0\r\no=- 1730624460385733 1 IN IP4 192.168.2.101\r\ns=Session streamed by \"preview\"\r\nt=0 0\r\na=tool:BC Streaming Media v202210012022.10.01\r\na=type:broadcast\r\na=control:*\r\na=range:npt=now-\r\na=x-qt-text-nam:Session streamed by \"preview\"\r\nm=video 0 RTP/AVP 96\r\nc=IN IP4 0.0.0.0\r\nb=AS:8192\r\na=rtpmap:96 H264/90000\r\na=fmtp:96 packetization-mode=1;profile-level-id=640033;sprop-parameter-sets=Z2QAM6wVFKCgPZA=,aO48sA==\r\na=recvonly\r\na=control:track1\r\nm=audio 0 RTP/AVP 97\r\nc=IN IP4 0.0.0.0\r\nb=AS:8192\r\na=rtpmap:97 MPEG4-GENERIC/16000\r\na=fmtp:97 streamtype=5;profile-level-id=1;mode=AAC-hbr;sizelength=13;indexlength=3;indexdeltalength=3;config=1408\r\na=recvonly\r\na=control:track2\r\nm=audio 0 RTP/AVP 0\r\na=control:track3\r\na=rtpmap:0 PCMU/8000\r\na=sendonly",
            "senders": [
                "0 PCMU/8000, bytes=0, receivers=1"
            ],
            "type": "RTSP active producer",
            "url": "rtsp://192.168.2.101/Preview_01_sub/",
            "user_agent": "go2rtc/1.9.2"
        }
    ]
}

**Reply by NickM-27** [reply]
You have a mic button so everything is working from the frigate side, if there is no doorbell sound that would be a go2rtc or doorbell issue
